tests/unit/local/apigw/test_local_apigw_service.py

```diff
@@ -590,3 +586,23 @@ def test_should_base64_encode_returns_true(self, test_case_name, binary_types, m
     ])
     def test_should_base64_encode_returns_false(self, test_case_name, binary_types, mimetype):
         self.assertFalse(LocalApigwService._should_base64_encode(binary_types, mimetype))
+
+
+class TestRouteEqualsHash(TestCase):
+
+    def test_route_in_list(self):
+        route = Route(function_name="test", path="/test", methods=["POST"])
+        routes = [route]
+        self.assertIn(route, routes)
+
+    def test_route_method_order(self):
```

Comment:
You should have eq tests for same object different data, same object, two objects same data, two objects different data, two different objects/types.